pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -1644,6 +1645,30 @@ func (gce *GCECloud) DeleteDisk(diskToDelete string) error {
 	return gce.waitForZoneOp(deleteOp, disk.Zone)
 }
 
+func (gce *GCECloud) GetPDLabels(name string) (map[string]string, error) {
```

Comment:
nit: Worth adding a comment here describing what this does, as it's not entirely clear from the signature. In particular it does not return all of the labels currently applied to the PD.  It returns the zone and region labels that we want to apply to the PD.  Also, the name param is the name of the PD, not the name of the label, or of anything else.